setup.py

```diff
@@ -51,7 +51,7 @@ class CustomInstallCommand(install):
     def run(self):
         install.run(self)
 
-        prefix = path.abspath(get_config_vars('prefix')[0])
+        prefix = path.abspath(get_config_vars('userbase')[0])
```

Comment:
Doesn't this break usual system-wide install?

Reply:
Fixed